language/testing-infra/transactional-test-runner/src/tasks.rs

```diff
@@ -23,21 +23,9 @@ pub enum RawAddress {
     Anonymous(AccountAddress),
 }
 
-fn parse_address_literal(s: &str) -> Result<AccountAddress> {
-    let (array, _) = move_compiler::shared::parse_address(s).ok_or_else(|| {
-        anyhow!(
-            "Failed to parse address {} to AccountAddress with Length {}",
-            s,
-            AccountAddress::LENGTH
-        )
-    })?;
-
-    Ok(AccountAddress::new(array))
-}
-
 impl RawAddress {
     pub fn parse(s: &str) -> Result<Self> {
-        if let Ok(addr) = parse_address_literal(s) {
+        if let Ok(addr) = AccountAddress::from_hex_fuzzy(s) {
```

Comment:
This isn't the same behavior. WIthout the `0x` parses from a decimal value, and that should be the behavior for this, and all CLI tools, to match the source language

Reply:
`parse_address_literal` was already doing fuzzy matching.  So, if that's the case I'm not making this any more wrong, but it was wrong in the first place

Reply:
This the wrong sort of fuzzy parsing though. It should be following the source languages convention: `0x` is hex, otherwise decimal

Reply:
But then again.... this is mostly for identifiers, so maybe we should just force the 0x

Should probably reserve the dec/hex parsing for args